From c43f856cc91c6adcb1b7d521bd03106c26767999 Mon Sep 17 00:00:00 2001
From: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Date: Wed, 14 May 2014 01:01:09 -0700
Subject: [PATCH v5 143/164] igb/ixgbe: remove return statements for void functions

Git-commit: c43f856cc91c6adcb1b7d521bd03106c26767999
Patch-mainline: v3.16-rc1
Reference: fate#317388, bsc#909491
Target: sle11-sp4

Remove useless return statements for void functions which do not need
it.

Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Tested-by: Phil Schmitt <phillip.j.schmitt@intel.com>
Tested-by: Aaron Brown <aaron.f.brown@intel.com>
Acked-by: Lee, Chun-Yi <jlee@suse.com>
---
 drivers/net/ethernet/intel/igb/e1000_nvm.c |    1 -
 1 file changed, 1 deletion(-)

Index: linux-3.0-SLE11-SP4/drivers/net/ethernet/intel/igb/e1000_nvm.c
===================================================================
--- linux-3.0-SLE11-SP4.orig/drivers/net/ethernet/intel/igb/e1000_nvm.c
+++ linux-3.0-SLE11-SP4/drivers/net/ethernet/intel/igb/e1000_nvm.c
@@ -797,5 +797,4 @@ etrack_id:
 		fw_vers->etrack_id = (eeprom_verh << NVM_ETRACK_SHIFT)
 			| eeprom_verl;
 	}
-	return;
 }
